Northstar.CustomServers/mod/scripts/vscripts/_mapspawn.gnut

```diff
@@ -199,6 +199,11 @@ var function CodeCallback_ClientCommand( entity player, array<string> args )
 		printl( key + " : " + value )
 	printl( "############################" )*/
 
+	if ( GetConVarBool("ns_should_log_all_clientcommands") )
```

Comment:
if you're going to do this outside of native, why not use the vanilla formatting that's commented out just above it?

Reply:
Well, for one thing, you can’t grep for it in server logs as easily because it’s across multiple lines.

But it’s also a bit needlessly verbose for just keeping a log of client commands, no?

Reply:
So Blue and I looked into doing it in native and it appears to be more convoluted than expected. Should we still do it in native or just merge this for now (with maybe a comment added about potentially doing it in native in the future) so that we can make next release sooner than later? ^^